apps/site/lib/site_web/templates/schedule/_trip_view.html.eex

```diff
@@ -1,4 +1,5 @@
 <%= SiteWeb.AlertView.group(alerts: @alerts, route: @route, date_time: @date_time, priority_filter: :high) %>
+<%= retirement_message(@conn) %>
```

Comment:
This might be simpler (you wouldn't need the `retirement_message` view function) as something like
```
<%= if @retirement_message do %>
  <%= SiteWeb.ViewHelpers.banner_message(@conn, :retirement_message) %>
<% end %>
```
(assuming the commuter rail part of the logic was already taken care of when `@retirement_message` was added to `conn.assigns`, which I think it should be)